src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
```suggestion
@Getter
@Setter
@Table(name....
```

Don't use `@Data` for entities